api/api/catalogue.go

```diff
@@ -62,7 +93,7 @@ func execRequest(
 		return
 	}
 
-	duration := time.Duration(request.SasDuration) * time.Minute
+	duration := time.Duration(request.SasDuration)
```

Comment:
That makes sense. Thanks for the explanation. I was unsure if the check against 0 makes sense without any units. I am mainly paranoid about `time.Duration` works internally and how it does comparisons. However, if we will allow only positive durations this will be probably more clear. :)

@ErlendHaa proposed positive times between 1ms and 24h for example. As mentioned above, this check could be included in the unmarshalling. In that case we would return a 400 code I assume. I can move this in a different issue/pull request.

Reply:
I just updated the check against for the presence of a SAS request to make sure that the same types are compared without any implicit conversion.

I would move the the check for positive durations into a separate issue if that is ok. Then we can also discuss what is the range of values that we want to accept and whether we do that in the unmarshalling or rather as a restriction within the type.

Reply:
I opened a new issue https://github.com/equinor/seismic-catalogue/issues/18 for checking the valid SAS durations.

The part with the implicit/explicit conversion I leave as it is.